Cinema system/Cinema system/Models/Utils/Response.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+
+namespace CinemaSystem.Models
+{
+    public class Response
+    {
+        public bool IsSuccessful { get; set; }
+        public List<string> ErrorDescriptions { get; set; }
+        
+        public Response()
+        {
+            ErrorDescriptions = new List<string>();
+        }
+    }
+
+    public class GetResponse<T> : Response
+    {
+        public T RequestedData { get; set; }
+    }
+
+    public class CreateResponse : Response
+    {
+        public int Id { get; set; }
+        public CreateResponse() : base()
```

Comment:
You don't need empty constructor, it will be created for you implicitly by the compiler